common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -846,9 +846,9 @@ class @Problem
     #   'enable' is a boolean to determine enabling/disabling of submit button.
     #   'changeText' is a boolean to determine if there is need to change the
     #    text of submit button as well.
-    attempts_remaining = @submitButton.data('attempts-remaining')
     if enable
-      if attempts_remaining != 0
```

Comment:
@alisan617 I remember you adding this data for a reason as part of a discussion.  What was that reason again?